src/Loading/babylon.sceneLoader.ts

```diff
@@ -1,16 +1,27 @@
 ﻿module BABYLON {
+    export interface ISceneLoaderPluginExtensions {
+        [extension: string]: {

```

Comment:
http://stackoverflow.com/questions/21070401/how-does-the-hash-variable-syntax-work-in-typescript